docs/documentation/unity/visualscripting/nodes.md

```diff
@@ -0,0 +1,143 @@
+# Visual Scripting
+
+Visual Scripting in Unity allows creating logic for virtual world applications using visual, node-based graphs without writing code. The Dolby.io Virtual World plugin is compatible with Unity Visual Scripting 2021. The plugin can be used with Visual Scripting and C# scripting at the same time. 
+
+Before using the plugin for visual scripting, make sure that you added the .NET SDK to Unity and initialized the SDK using the [Initializing](../unity.md#initialization) procedure.
+
+### Nodes
+
+Nodes are the most basic part of scripts in Visual Scripting. A node can listen for events, get the value of a variable, modify a component on a GameObject, and more.
+Once the plugin is successfully installed, the Dolby.io nodes are accessible in the `Add Node` contextual menu, under the DolbyIO category:
+<div style="text-align:left">
+    <img style="padding:25px 0" src="~/images/nodes/menu.png" width="500px">
+</div>
+
+---
+### Initialize
+
+Initialize the SDK and connects to the Dolby.io platform. In your production application deployment, please follow our security best practice [here](https://docs.dolby.io/communications-apis/docs/guides-client-authentication) to setup a server through which you can acquire a temporary client access token. 
+<div style="text-align:left">
+    <img style="padding:25px 0" src="~/images/nodes/init.png" width="300px">
+</div>
+
+| Name  | Direction | Type | Description  |
+|---|:---|:---|:---|
+| **Access Token** | Input | String| The [access token](xref:DolbyIO.Comms.DolbyIOSDK.InitAsync(System.String,DolbyIO.Comms.RefreshTokenCallBack)) provided by the customer's backend. |
```

Comment:
This will change a bit with my last PR for visual scripting. Access Token is now either a string or a function that returns a string.